pulp_2to3_migration/app/plugin/content.py

```diff
@@ -232,7 +232,8 @@ def get_remote_by_importer_id(importer_id):
             lazy_wo_catalog = is_lazy_type and not pulp2lazycatalog
             if pulp2content.pulp3_content is not None and (not is_lazy_type or lazy_wo_catalog):
                 if pb:
-                    pb.increment()
+                    pb.total -= 1
+                    pb.save()
```

Comment:
This is for the sake of not showing counters for the skipped content, otherwise, when you remigrate with no changes, it's the only counter for migration which is not 0.

Reply:
+1

Reply:
+1, i meant to -1 instead of increment, that was my oversight.